filtres/reply.py

```diff
@@ -27,6 +27,8 @@ def buscar_ticket_id(self, string, regex):
             logger.info("Trobat ticket %s" % ticket_id)
             return ticket_id
         except Exception as e:
+            logger.info(e)
+            logger.info("Error found")
```

Comment:
```suggestion
            logger.info("Error found: %s" % e)
```
Per què no poses el missatge en una sola línia com a sobre?

Reply:
Perquè m'ha passat per alt. Ho canvio.